quantities/quantity.py

```diff
@@ -120,9 +120,9 @@ def __new__(cls, data, units='', dtype=None, copy=True):
                 data = data.rescale(units)
             if isinstance(data, unit_registry['UnitQuantity']):
                 return 1*data
-            return np.array(data, dtype=dtype, copy=copy, subok=True).view(cls)
+            return np.asanyarray(data, dtype=dtype).view(cls)
 
-        ret = np.array(data, dtype=dtype, copy=copy).view(cls)
+        ret = np.asarray(data, dtype=dtype).view(cls)
```

Comment:
After these changes we are no longer using the `copy` keyword argument passed into `__new__`. I contemplated dropping it, but I fear that the change would be too disruptive for backwards compatibility. Perhaps changing its default to `None` and conditionally issuing a DeprecationWarning when it is set to `True`/`False` is a viable option? And the deprecation warning would inform the user that the copy-kwargs will be dropped in future release of `quantities`.